bin/build_ext_debs

```diff
@@ -217,9 +216,22 @@ package_python_deb() {
             --disable-pip-version-check \
             --no-deps $pip_pkg_name
 
-        tarball=$tar_pkg_name*.tar.gz
+		if [[ $pip_pkg_name == *".git"* ]]
```

Comment:
Is this pyformance specific? That might be obvious later.  Maybe more specific match or a comment.

Reply:
Fair point since this is a "quick fix" I've changed it check for "pyformance" instead